encoding/protobuf/outbound_test.go

```diff
@@ -22,18 +22,25 @@ package protobuf
 
 import (
 	"context"
+	"io/ioutil"
 	"testing"
 
+	"github.com/gogo/protobuf/jsonpb"
+	"github.com/gogo/protobuf/proto"
 	"github.com/golang/mock/gomock"
+	"github.com/golang/protobuf/ptypes"
```

Comment:
Hmm, do you mind trying `gogo/protobuf/types` here instead?

Reply:
yup, nice catch, I was leaning on my `goimports`